content/manager/manager-upgrade.md

```diff
@@ -0,0 +1,219 @@
+---
```

Comment:
before reviewing, please change the name of the file to `upgrade.md`. We're already in the context of `manager`

And please also change the title to `Upgrading to a newer version`.